vagrant/.gitignore

```diff
@@ -1 +1,3 @@
-.vagrant
\ No newline at end of file
+.vagrant
+*.box
+Berksfile.lock
```

Comment:
Actually, no, that's generally not a good idea, due to how Berkshelf does cookbook version resolution. If you really feel the cookbook versions should be tightly pinned, that should be done in `metadata.rb` or `Berksfile`.